rgw/admin/bucket.go

```diff
@@ -112,7 +112,7 @@ func (api *API) GetBucketInfo(ctx context.Context, bucket Bucket) (Bucket, error
 	return ref, nil
 }
 
-// GetBucketPolicy - http://docs.ceph.com/docs/mimic/radosgw/adminops/#get-bucket-or-object-policy
+// GetBucketPolicy - http://docs.ceph.com/en/mimic/radosgw/adminops/#get-bucket-or-object-policy
```

Comment:
Since we are changing this line, should we link `latest` instead of `mimic` too?

Reply:
fixed